app/helpers/application_ui_helper.rb

```diff
@@ -0,0 +1,31 @@
+module ApplicationUiHelper
+  # Create a edit button with icon
+  #
+  # @param path [String] The path of the link.
```

Comment:
Actually should be "not that suitable", i.e. when the  resources is non-restful. Also we need to specify actions.

path simply works for any cases.